airbyte-integrations/connectors/source-iterable/unit_tests/test_source.py

```diff
@@ -2,10 +2,35 @@
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
 
+from unittest.mock import patch
+
+import pytest
+from source_iterable.api import Lists
 from source_iterable.source import SourceIterable
 
 
+@pytest.fixture(name="config")
+def config_fixture():
```

Comment:
If this fixture is reused in multiple test module please declare it in `conftest.py`, it will avoid code duplication and allow all modules to access it.